src/target/llvm/codegen_llvm.cc

```diff
@@ -309,6 +309,9 @@ llvm::Type* CodeGenLLVM::DTypeToLLVMType(const DataType& dtype) const {
       default:
         LOG(FATAL) << "do not support " << dtype;
     }
+  } else if (dtype.is_bfloat()) {
+    CHECK_EQ(dtype.bits(), 16);
```

Comment:
Since bfloat is assumed to be 16bit, can we keep the terminology more consistent? Since the data type is termed as `bf`, `bf16`, `bfloat16`, `bfloat` in the proposed change. Or are we going to support more data types like bfloat18 and bfloat20 in the future?

Reply:
Sorry for the inclarity. I think in bfloat[X], only X=16 makes sense. But TVM's type system allows specifying the `bits` of a type. So here is the checking to make sure it is bf16.

Reply:
A good question. Will we treat TensorFloat-32 as bfloat20? If so, then `bits` is useful to distinguish those.